packages/entity-list/src/components/ListView/ListView.js

```diff
@@ -53,8 +53,12 @@ class ListView extends React.Component {
             this.props.formDefinition && this.props.formDefinition.children.map(child => {
               if (child.componentType === form.componentTypes.TABLE) {
                 return <ListWrapper searchFormPosition={this.props.searchFormPosition} key={`tableWrapper-${child.id}`}>
-                  <TableContainer key={`table-${child.id}`}
-                    columnDefinitions={getColumnDefinition(child, this.props.parent)}/>
+                  <TableContainer
+                    key={`table-${child.id}`}
+                    columnDefinitions={
+                      getColumnDefinition(child, this.props.sorting, this.props.parent, this.props.intl)
```

Comment:
(there is a `props` variable defined on line 45. so this line could be a little shorter. I personally like destructuring the props on the first line of the render method for even less code an often times improved readability on the following lines)